codegen/codegen_outofplacebatching.py

```diff
@@ -247,17 +188,8 @@ def __init__(self, schema, ops):
 
     schemas = get_signatures(args.path, include_op=True)
     schemas = [schema for schema in schemas if is_schema_outplace(schema)]
-    unique_schemas = {}
-    for schema in schemas:
-        op, *schema = schema
-        unique_id = get_hash(schema)
-        if unique_id in unique_schemas:
-            unique_schemas[unique_id].ops.append(op)
-        else:
-            unique_schemas[unique_id] = Container(schema, [op])
-
-    codes = [lower(*container.schema, i, container.ops)
-             for i, container in enumerate(unique_schemas.values())]
+    codes = [gen_plumbing(*schema) for schema in schemas]
+    print("#pragma once")
     print("#include <functorch/csrc/OutOfPlacePlumbing.h>")
```

Comment:
@zou3519 shouldn't we remove this line as generated file (functorch/csrc/OutOfPlacePlumbing.h) includes itself ?

Reply:
uh oh, good catch

Reply:
pragma once should help to avoid inf loop